### PR TITLE
Doc(eos_cli_config_gen): Include docs for router segment-security

### PR DESCRIPTION
**Pull request by jonxstill**
## Change Summary

Fix missing documentation table for router segment-security.

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
Include router-segment-security.md table in input-variables.md

## How to test
Verify Router segment-security section appears in docs for `eos_cli_config_gen`.

### Repository Checklist

- [X] My code has been rebased from devel before I start
- [X] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [X] My change requires a change to the documentation and documentation have been updated accordingly.
- [X] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md [hunk 1]
```diff
@@ -905,6 +905,12 @@ roles/eos_cli_config_gen/docs/tables/router-ospf.md
 roles/eos_cli_config_gen/docs/tables/router-path-selection.md
 --8<--
 
+### Router segment-security
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md]: This is under the level-2 header of "Routing". I think we should find a better "home" for it.

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md]: see line 956 `## Security`

Comment (jonxstill) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md]: Agreed - will fix.

**Review by carlbuchmann — APPROVED** [review]
LGTM - thanks for fixing this @jonxstill

**Pull request merged**